src/xcode/ENA/ENA/Source/Services/Risk/Provider/Model/RiskProvidingConfiguration.swift

```diff
@@ -68,3 +68,19 @@ extension RiskProvidingConfiguration {
 		return shouldPerformExposureDetection(lastExposureDetectionDate: detectionDate) ? .possible : .waiting
 	}
 }
+
+enum ExposureDetectionContext {
+	case rateLimitLogger
+	case other
+}
+
+extension ExposureDetectionContext {
+	var description: String {
+		switch self {
+		case .rateLimitLogger:
+			return " for rate limit logging"
```

Comment:
```suggestion
			return "For rate limit logging"
```

Reply:
@naveeddotio in the current implementation, this "description" is directly used in the log message of RiskProvidingConfiguration, so I need it like this (leading blank and no caps). Should I add a comment for less confusion? Or a different name for that var?

Reply:
Perhaps use `loggingExtension` as var name and add a small comment that the text is concatenated directly into a log message?

Reply:
Done 😁